android/gradle.properties

```diff
@@ -1,4 +1,4 @@
 org.gradle.jvmargs=-Xmx1536M
-android.enableR8=true
 android.useAndroidX=true
 android.enableJetifier=true
+android.enableR8=true
```

Comment:
Was this change necessary or is it accidental?

Reply:
it was giving warning in android studio, so I remove that but later on, it was added again by studio.